dispatcher/src/conf.d/rewrites/rewrite.rules

```diff
@@ -18,6 +18,10 @@ RewriteRule ^(.*)$ https://wknd.site%{REQUEST_URI} [R=301,L,E=cdncache]
 RewriteCond %{REQUEST_URI} ^/?$
 RewriteRule ^(/)$ /us/en.html [R=301,L,E=cdncache]
 
+# CORS - send 204 for CORS requests which are not trusted
+RewriteCond expr "reqenv('CORSProcessing') == 'true' && reqenv('CORSTrusted') == 'false'"
+RewriteRule "^(.*)" - [R=204,L]
+
```

Comment:
This rewrite rule will not work in its current form/position, as `conf.d/rewrites/default_rewrite.rules` contains `rewrite` at as a last rule with flag `[PT]`, and that implicitly implies `L` flag. See PT section on https://httpd.apache.org/docs/2.4/rewrite/flags.html

Reply:
Jabran, have you tested the PR?

Reply:
Yes, tested this using a local author-publish-dispatcher setup.